S/SDL2/build_tarballs.jl

```diff
@@ -3,37 +3,47 @@
 using BinaryBuilder, Pkg
 
 name = "SDL2"
-version = v"2.24.2"
+version = v"2.32.10"
 
 # Collection of sources required to build SDL2
 sources = [
     GitSource("https://github.com/libsdl-org/SDL.git",
-              "55b03c7493a7abed33cf803d1380a40fa8af903f"),
+              "5d249570393f7a37e037abf22cd6012a4cc56a71"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/SDL*/
-mkdir build && cd build
+cd $WORKSPACE/srcdir/SDL*
+
 FLAGS=()
 if [[ "${target}" == *-linux-* ]] || [[ "${target}" == *-freebsd* ]]; then
     FLAGS+=(--with-x)
     if [[ "${target}" == *-freebsd* ]]; then
         # Needed for libusb_* symbols
         export LIBUSB_LIBS="-lusb"
     fi
-elif [[ "${target}" == aarch64-apple-* ]]; then
+elif [[ "${target}" == *-apple-* ]]; then
     # Link to libclang_rt.osx to resolve the symbol `___isPlatformVersionAtLeast`:
     # <https://github.com/libsdl-org/SDL/issues/6491>.
     export LDFLAGS="-L${libdir}/darwin -lclang_rt.osx"
 fi
+if [[ "${target}" == riscv64-* ]]; then
+    # Newer compilers report a warning which is an error
+    FLAGS+=(CFLAGS=-Wno-error=incompatible-pointer-types)
+fi
+if [[ "${target}" == *musl* ]]; then
+    export LDFLAGS=-liconv
+fi
+
+mkdir build && cd build
 ../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
     --enable-shared \
     --disable-static \
     "${FLAGS[@]}"
+
 make -j${nproc} V=1
 make install V=1
-install_license ../LICENSE.txt
+cd ..
```

Comment:
```suggestion
```
?

Reply:
But also, why removing the `install_license` command?  Does it hurt?

Reply:
The `cd` is necessary for BinaryBuilder to find the license. It doesn't look up.